Library/brew.rb

```diff
@@ -25,18 +25,30 @@
   exit 0 if ARGV.length == 1
 end
 
-# Check for bad xcode-select before anything else, because `doctor` and
-# many other things will hang
-# Note that this bug was fixed in 10.9
-if OS.mac? && MacOS.version < :mavericks && MacOS.active_developer_dir == "/"
-  odie <<-EOS.undent
-  Your xcode-select path is currently set to '/'.
-  This causes the `xcrun` tool to hang, and can render Homebrew unusable.
-  If you are using Xcode, you should:
-    sudo xcode-select -switch /Applications/Xcode.app
-  Otherwise, you should:
-    sudo rm -rf /usr/share/xcode-select
-  EOS
+# Check for bad xcode-select before other checks, because `doctor` and
+# many other things will hang. Note that this bug was fixed in 10.9
```

Comment:
Yes, but I can fix that on merge after the CI does its thing.

Sent from my iPhone

> On 15 Oct 2015, at 15:24, Martin Afanasjew <notifications@github.com> wrote:
> 
> In Library/brew.rb:
> 
> > @@ -25,18 +25,30 @@
> >    exit 0 if ARGV.length == 1
> >  end
> >  
> > -# Check for bad xcode-select before anything else, because `doctor` and
> > -# many other things will hang
> > -# Note that this bug was fixed in 10.9
> > -if OS.mac? && MacOS.version < :mavericks && MacOS.active_developer_dir == "/"
> > -  odie <<-EOS.undent
> > -  Your xcode-select path is currently set to '/'.
> > -  This causes the `xcrun` tool to hang, and can render Homebrew unusable.
> > -  If you are using Xcode, you should:
> > -    sudo xcode-select -switch /Applications/Xcode.app
> > -  Otherwise, you should:
> > -    sudo rm -rf /usr/share/xcode-select
> > -  EOS
> > +# Check for bad xcode-select before other checks, because `doctor` and
> > +# many other things will hang. Note that this bug was fixed in 10.9
> Shouldn't this comment be inside the if OS.mac? block, right before if MacOS.version < :mavericks && MacOS.active_developer_dir == "/"? Otherwise, it gives the impression of applying to both of the following two checks.
> 
> —
> Reply to this email directly or view it on GitHub.
>